### ISSUE TITLE
Лишние css правила

### ISSUE THREAD

**Issue by Maksimus-Prime** [rustho/TaskFSD-2UI#62]
Есть лишние правила, к примеру элементу с display: (block | flex) задается ширина 100%, это говорит о том что ты не знаешь как работают display: (block | flex)
Проходил html академию? если да, то очень подозрительно что ты не знаешь таких базовых вещей :) ищи статейки, читай как ведут себя display: inline, inline-block, block, flex и т.д., и опиши здесь вкратце ключевые характеристики каждого из display

**Comment by rustho** [comment]
Эээ... ну начнем с того что это в теории ничего не меняет (((: 
да блок - априори растягивается на всю ширину, но я ставил длину вручную ибо блок не растягивался сразу. Гляну, попробую убрать. 
блок - растягивается на всю ширину, и обрывает содержимое перед и после блока
инлайн - это грубо говоря элемент строки, встраивается в блок, ничего не обрывает, ширина и высота по содержимому
инлайн-блок нечто среднее между ними. элемент строки, встраивается в блок, ничего не обрывает, но может иметь кастомную ширину высоту
флекс - это блок, но внутренние блоки становятся сами по себе резиновыми
грид - тоже блок, но внутренние блоки растягиваются по пропорциям, что-то типо флексов на максималках.

**Comment by Maksimus-Prime** [comment]
Правило из стандартов
!Удалять все неиспользуемые правила и правила, которые никак не влияют на отображение (например, дублирующие свойства по умолчанию или нулевые margin, если они и так не выставлены) - они могут в дальнейшем при правках внести неочевидное поведение и из-за них тяжело разбираться в проекте
-------------------------------
Так если бы ты все нормально сделал бы, то не пришлось бы прописывать такие вещи :)
Почему у тебя внешняя блока задана у блоков? там же в риззоме был долгая дисскусия про позиционирование блока
![2018-03-17 12-02-32](https://user-images.githubusercontent.com/27883524/37551835-1f9226de-29db-11e8-9fce-42a285d53fa5.png)

**Comment by rustho** [comment]
Хм, ну "Есть лишние правила, к примеру элементу с display: (block | flex) задается ширина 100%" этого бы хватило. 
"внешняя блока задана у блоков"? - если ты имеешь в виду что я должен был обернуть в родительский контейнер, то это он и есть. сам прогрессбар - ui-progressbar.

**Comment by Maksimus-Prime** [comment]
А вот этот "родительский контейнер", что это за сущность по бэму?

**Comment by rustho** [comment]
блок

**Comment by Maksimus-Prime** [comment]
так почему ты блоку задаешь внешнюю геометрию?

**Comment by rustho** [comment]
А дак, так бы и спросил сразу насчет внешней геометрии, понял, переделаю

**Comment by Maksimus-Prime** [comment]
Нет никакого контейнера, просто навешал еще один класс, который так же лезет внутрь блока и мутирует его

**Comment by Maksimus-Prime** [comment]
Нет никакого контейнера, просто навешал еще один класс, который так же лезет внутрь блока и мутирует его

**Comment by rustho** [comment]
Ой, щет ((((: исправлю (:

**Comment by Maksimus-Prime** [comment]
:)

**Comment by Maksimus-Prime** [comment]
1. Что за странный модификатор wrapper_type_fixwidth600?
![2018-03-27 19-03-48](https://user-images.githubusercontent.com/27883524/37966111-a4656a74-31f1-11e8-91d5-cb0b097284ca.png)
2. Зачем div задается width: 100%?
![2018-03-27 19-05-41](https://user-images.githubusercontent.com/27883524/37966224-045cf0be-31f2-11e8-82b9-d40dcc406567.png)
![2018-03-27 19-05-58](https://user-images.githubusercontent.com/27883524/37966225-048a5ebe-31f2-11e8-9702-93fa894dc156.png)
![2018-03-27 19-06-23](https://user-images.githubusercontent.com/27883524/37966226-04d6f24c-31f2-11e8-866c-fbda7d83c3ea.png)

**Comment by rustho** [comment]
поправил

**Comment by Maksimus-Prime** [comment]
Все равно не понятно, зачем модификатор (1) ?

**Comment by rustho** [comment]
ну чтобы всё не разбредалось 
![image](https://user-images.githubusercontent.com/26579566/38013709-50a1d9fa-3290-11e8-98d0-598d218f166b.png)

**Comment by Maksimus-Prime** [comment]
А по другому это никак не задать?

**Comment by rustho** [comment]
Хм... ну потому что я модифицирую враппер... просто во враппер смысла задавать нет, некрасиво и не понятно, а так по верстке сразу видно что этот враппер фиксированной длины

**Comment by Maksimus-Prime** [comment]
Нет, это нормальная ситуация когда у тебя есть не семантические конструкции, которые ты юзаешь для создания сетки :) и задать ей ширину в 594px это тоже нормально

**Comment by rustho** [comment]
Омг, ну ладно (:
